nixos/modules/services/networking/wireguard.nix

```diff
@@ -234,14 +242,16 @@ let
       keyToUnitName = replaceChars
```

Comment:
can't we use `escapeSystemdPath` here?

Reply:
This code is not part of the PR and I'd rather not touch things that don't need to be.